pyproject.toml

```diff
@@ -55,20 +59,13 @@ testing = [
 ]
 pre_commit = [
     "pre-commit",
-    "pylint~=2.12.2",
-    "mypy==0.930",
+    "pylint~=3.3.1",
+    "mypy==1.13",
     "types-setuptools==65.7.0.3",
     "types-PyYAML",
 ]
 dev = [
-    "sphinx",
-    "sphinx-rtd-theme",
-    "pgtest~=1.3.1",
-    "aiida-diff",
-    "pytest-datadir",
-    "pre-commit",
-    "pylint~=2.12.2",
-    "mypy==0.930",
+    'aiida-testing[testing,pre_commit,docs]',
```

Comment:
Self-referencing extras are allowed in `pyproject.toml` (but not in `setup.cfg`)